src/components/NewGalaxyModal.js

```diff
@@ -186,59 +207,118 @@ export const NewGalaxyModal = _.flow(
     ])
   }
 
-  const getEnvMessageBasedOnStatus = isTitle => {
-    const waitMessage = isTitle ? '' : 'This process will take up to a few minutes.'
+  const getEnvMessageBasedOnStatus = app => {
+    const waitMessage = 'This process will take up to a few minutes.'
 
     return !app ?
-      isTitle ? 'Cloud environment' : 'Environment will consist of an application and cloud compute.' :
+      'consists of application configuration, cloud compute and persistent disk(s).' :
       Utils.switchCase(app.status,
-        ['STOPPED', () => `Cloud environment is now paused ${!isTitle ? '...' : ''}`],
-        ['PRESTOPPING', () => 'Cloud environment is preparing to stop.'],
-        ['STOPPING', () => `Cloud environment is pausing. ${waitMessage}`],
-        ['PRESTARTING', () => 'Cloud environment is preparing to start.'],
-        ['STARTING', () => `Cloud environment is starting. ${waitMessage}`],
-        ['RUNNING', () => 'Environment consists of an application and cloud compute.'],
+        ['STOPPED', () => `is paused.`],
```

Comment:
I see `pause`, `stop`, `start`, and `resume` here. Can we use consistent language?

Reply:
Totally agreed - especially important for the user-facing ones. I've updated some things in my PR as I touched them but we have a [ticket in progress](https://broadworkbench.atlassian.net/browse/IA-2539) to go through the codebase for it.

Reply:
For clarity, did you just mean the "is ..." text itself? Or do you mean, the statuses themselves?

Reply:
I was thinking we rename the statuses as well since they're user-facing too; i.e.

```javascript
['RESUMING', () => `Cloud environment is starting. ${waitMessage}`],
['PAUSED', () => `is paused.`]
```

What do you think?

Reply:
I think this suggestion is good. Will look at the best way to accomplish this. Feels like we should have a helper function to make this work, might tackle this in its own PR.